src/shared-components/dropdown/mobileDropdown.js

```diff
@@ -23,7 +23,12 @@ const MobileDropdown = ({
         }
 
         return (
-          <option key={option.value} value={option.value} disabled={isDisabled}>
+          <option
```

Comment:
looks like this element supports disabled natively as well
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option